reference/features/configuration.md

```diff
@@ -32,342 +30,351 @@ Server is listening on: http://127.0.0.1:7700
 
-**Default value**: `None`
+If no master key is provided in a `production` environment, MeiliSearch will throw an error and refuse to launch.
+
+[Learn more about MeiliSearch's use of security keys.](/reference/features/authentication.md)
 
 ### Analytics
 
 **Environment variable**: `MEILI_NO_ANALYTICS`
 **CLI option**: `--no-analytics`
-
-Deactivates analytics.
-
-Analytics allow us to know how many users are using MeiliSearch and the following:
-
-- The application version.
-- The environment: development/production.
-- The number of days since the first start: Segment development/production.
-- The user email: (*Optional*) If the user wants to have alerts.
-- The server provider: (*Optional*).
-- The database size.
-- The last update time.
-- The number of updates.
-- The number of documents per index.
-
 **Default value**: `false`
 
-### Environment
-
-**Environment variable**: `MEILI_ENV`
-**CLI option**: `--env`
-
-By default, MeiliSearch runs in `development` mode.
-
-- `production`: the [master key](/reference/features/authentication.md) is **mandatory**.
-- `development`: the [master key](/reference/features/authentication.md) is **optional**, and logs are output in "info" mode (_console output_).
-
-If the server is running in development mode more logs will be displayed, and the master key can be avoided which implies that there is no security on the updates routes.
-This is useful to debug when integrating the engine with another service.
-In production mode, the [web interface](/reference/features/web_interface.md#web-interface) is disabled.
-
-**Default value**: `development`
-
-### Payload limit size
-
-**Environment variable**: `MEILI_HTTP_PAYLOAD_SIZE_LIMIT`
-**CLI option**: `--http-payload-size-limit`
-
-The maximum size, in bytes, of accepted JSON payloads.
-
-**Default value**: `104857600` (+=100MB)
-
-### SSL authentication path
-
-**Environment variable**: `MEILI_SSL_AUTH_PATH`
-**CLI option**: `--ssl-auth-path`
-
-Enable client authentication, and accept certificates signed by those roots provided in CERTFILE
+Deactivates MeiliSearch's built-in analytics and telemetry when set to `true`.
 
-**Default value**: `None`
-
-### SSL certificates path
+By default, MeiliSearch collects the following data from all instances that do not explicitly opt-out:
 
-**Environment variable**: `MEILI_SSL_CERT_PATH`
-**CLI option**: `--ssl-cert-path`
+- Application version
+- Environment (development or production)
+- Number of days since the first start (segment development/production
+- Database size
+- Last update time
+- Number of updates
+- Number of documents per index
 
-Read server certificates from CERTFILE. This should contain PEM-format certificates in the right order (the first certificate should certify KEYFILE, the last should be a root CA).
+If a user does not disable analytics, they may also provide an email address and their server provider by setting the environment variables `MEILI_USER_EMAIL` and `MEILI_SERVER_PROVIDER`.
```

Comment:
Yeah, this is a really gnarly bit in pretty much every sense ("not disabled", only exists as an env var, is of questionable value). People can give us their e-mails out of the goodness of their hearts, but I'm not sure we actually do anything with them.

I included these options for the sake of completeness, but looking at it now I wonder if it's not a net win for everyone if we just remove this paragraph and leave the "functionality" undocumented. 

@gmourier, what do you think?

Reply:
I have removed this paragraph. `MEILI_USER_EMAIL` and `MEILI_SERVER_PROVIDER` add noise for little benefit since none of our users seem to use them (understandably).